cloudfunctions2_basic_auditlogs/main.tf

```diff
@@ -4,8 +4,12 @@
 # and the docs:
 # https://cloud.google.com/eventarc/docs/path-patterns
 
+resource "random_id" "prefix" {
+  byte_length = 8
+}
+
 resource "google_storage_bucket" "source-bucket" {
```

Comment:
can we scrub dashes from any resources names and replace with `_`

Reply:
Opened an issue since this line was not changed in this PR.